### ISSUE TITLE
Have set anonymousdse correctly but still unable to query sample config file LDAP

### ISSUE THREAD

**Issue by mchilson** [glauth/glauth#320]
I am running glauth on windows and have the latest binary

When I do a connection setup through JXExplorer:
![2023-07-06_10-58](https://github.com/glauth/glauth/assets/4140991/146b2b34-9682-4aae-abff-3616d64d8bab)

I get the following error:

Thu, 06 Jul 2023 10:55:13 -0500 INF handleSearchRequest error LDAP Result Code 50 "Insufficient Access Rights": Search Error: Anonymous BindDN not allowed

Also, when trying to login with any of the users in the config file I get the following error:

Thu, 06 Jul 2023 11:06:26 -0500 INF Bind request basedn=dc=glauth,dc=com binddn=serviceuser src=127.0.0.1:53719
Thu, 06 Jul 2023 11:06:26 -0500 INF BindDN not part of our BaseDN basedn=dc=glauth,dc=com binddn=serviceuser

My config file:
See attached zip file

[glauth.cfg.zip](https://github.com/glauth/glauth/files/11970156/glauth.cfg.zip)

Any ideas?

**Comment by mchilson** [comment]
Please disregard. I just found an Issue about anonymous not currently being implemented. It would be a nice feature to add. Thanks!

**Comment by cfr-convoso** [comment]
Yes. There is a setting to allow anonymous binds to retrieve the root DSE prior to actual authenticated access. Beyond that, the code rejects queries.
Can you expand on your use case?

**Comment by mchilson** [comment]
I was mistaken about how it was supposed to work. I was just testing the LDAP server. Thanks for taking the time to respond. Many thanks!
